tinkergraph-gremlin/src/main/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerVertex.java

```diff
@@ -43,17 +36,40 @@
 public final class TinkerVertex extends TinkerElement implements Vertex {
 
     protected Map<String, List<VertexProperty>> properties;
-    protected Map<String, Set<Edge>> outEdges;
-    protected Map<String, Set<Edge>> inEdges;
-    private final TinkerGraph graph;
+    protected Map<String, Set<Object>> outEdges;
```

Comment:
Do the edge maps need to be relaxed to Object types here?

Reply:
yes, because map stores edge id

Reply:
Can we document this in the code? I assume this is related to that test case change? Why was it necessary?